requirements/base.in

```diff
@@ -5,8 +5,10 @@ Django
 requests==2.22.0
 djangorestframework
 django-rest-swagger==2.1.2
-django-haystack==2.8.0
-elasticsearch>=1.0.0,<2.0.0
+elasticsearch>=7.8,<8.0
+elasticsearch-dsl>=7.2,<8.0
+django-elasticsearch-dsl>=7.1,<8.0
+django-elasticsearch-dsl-drf
```

Comment:
We didn't add any limits, does it ok?

Reply:
I think yes. The reason why I did not set the pinned version for `django-elasticsearch-dsl-drf`, is that everything is in the `base.in` file. That means `pip compile` will be launched. We have `django-elasticsearch-dsl>= 7.1,<8.0`, where the version is explicitly specified. For the compile version for django-elasticsearch-dsl-drf has to be a suitable version for `django-elasticsearch-dsl> = 7.1, <8.0` should be installed since this is its main(parent) dependency. Thinking it is ok.

Reply:
Ok! Let it be